grafana-backup/dashboard-exporter.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
All tools should have docstring. Said that, could you please try to write this tool in Go. The code can be smaller, more portable and easy to maintain than Python.